bats/compatibility/runner.sh

```diff
@@ -64,8 +64,12 @@ function test_dolt_version() {
   echo testing "$ver" at "$bin"
   PATH="`pwd`"/"$bin":"$PATH" setup_repo "$ver"
 
+# Changes to the NBS manifest broke forward compatibility when multiple clients with different
+# versions interact with the same repo instance. However, in a more realistic setting of
+# clients interacting with a repo through a remote, compatibility is not broken.
+# todo: update compatibility tests to have different client versions interact through a remote.
```

Comment:
There are two types of "remotes" – one is `doltremoteapi`, where the comment text seems to be true, and one is where each client actually has write access to the underlying storage assets, and in particular the manifest. Both `file:///` and `aws:` remotes are the second type. This comment isn't true for those remotes...